pex/variables.py

```diff
@@ -33,11 +33,34 @@ def iter_help(cls):
       variable_type, variable_text = cls.process_pydoc(getattr(value, '__doc__'))
       yield variable_name, variable_type, variable_text
 
+  @classmethod
+  def from_rc(cls, rc):
+    """Read pex runtime configuration variables from a pexrc file.
+
+       :param rc: an absolute path to a pexrc file
+       :return ret_vars: a dict object containing key values pairs found in processed pexrc files
```

Comment:
these and docstring continuations should indent to align with the opening `"""`, e.g.

```
"""The docstring summary which might
spill onto a new line.

Extended summary would go here.

:param etc: Would go here.
"""
```